src/sio.ts

```diff
@@ -117,6 +146,56 @@ export class RPSIO {
       case GPIO_HI_OE_XOR:
         this.qspiGpioOutputEnable ^= value & GPIO_MASK;
         break;
+      case DIV_UDIVIDEND:
+        this.divDividend = value >>> 0;
+        this.divCSR = 0b10;
```

Comment:
what's the reason for updating divCSR twice in the same switch case?

Reply:
I was waiting for You on this...
well the datasheet states that the CSR lsb is cleared at the beginning of the calculation and set once result is ready (after 8 cpu cycles).
On a single core this assignment does not make any sense, since I don't know how multicore will be implemented it could be something that will matter in case of concurrency against hardware divider.

Reply:
That's a good point. When we get to multicore, we'll just execute one instruction at a time (so jumping back and forth between running core0 and core1 code). 

In any case, JavaScript does not support multi-threading natively. The only way is using Web Workers (or [worker threads](https://nodejs.org/api/worker_threads.html) on Node.js). But even then, the only way two threads can access the same memory is using a SharedArrayBuffer, so if we decide at some point to truly run this on two threads (and share the divider memory between them) we'll have to change all these variables to be stored inside a SharedArrayBuffer and refactor the the code, and I don't see it happening any time soon.

So to summarize, I think we can safely remove the `this.divCSR = 0b10;`

Reply:
note that SIO dividers/interpreters are core local

Reply:
That's a good point to remember when we implemented multi-core. I noted this!